ecs-cli/modules/cli/local/project/project.go

```diff
@@ -29,7 +29,24 @@ import (
 	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/commands/flags"
 	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/config"
 	"github.com/aws/aws-sdk-go/service/ecs"
+	composeV3 "github.com/docker/cli/cli/compose/types"
 	"github.com/urfave/cli"
+	yaml "gopkg.in/yaml.v2"
+)
+
+// TODO: Those three blocks of constants below should be put into another place
+const (
+	// taskDefinitionLabelType represents the type of option used to
+	// transform a task definition to a compose file e.g. remoteFile, localFile.
+	// taskDefinitionLabelValue represents the value of the option
+	// e.g. file path, arn, family.
+	taskDefinitionLabelType  = "ecsLocalTaskDefType"
```

Comment:
Can we change the value of these constants to be similar to [SecretLabelPrefix](https://github.com/aws/amazon-ecs-cli/pull/806/files#diff-03f22076bccfd316ec476300c12e1a64R46)?

So something like `ecs-local.task.type` and `ecs-local.task.value`. I think this makes it obvious that all labels that start with `ecs-local.` were created by us.

Reply:
Yea, I agree with you